code/game/objects/items/fireaxe.dm

```diff
@@ -88,3 +88,22 @@
 /obj/item/fireaxe/metal_h2_axe/ComponentInitialize()
 	. = ..()
 	AddComponent(/datum/component/two_handed, force_unwielded=5, force_wielded=23, icon_wielded="[base_icon_state]1")
+
+/obj/item/fireaxe/metal_h2_axe/update_icon_state()
```

Comment:
Why are these update_icon_state overrides neccessary ? They seem identical to parent.

Reply:
you're right, i'll remove it from both the metal_h2_axe and the elder_atmosian_fireaxe